### PR TITLE
Update CefGlue to fix memory leaks

### PR DESCRIPTION
**Pull request by brunocunhasilva**
Update CefGlue to a version that addressed a few leaks in that component.

Did some minor improvements to the code.

### CODE REVIEW

File: WebViewControl/WebView.cs [hunk 1]
```diff
@@ -163,28 +191,33 @@ public partial class WebView : IDisposable {
         }
 
         public void Dispose() {
-            InnerDispose();
+            InnerDispose();
             GC.SuppressFinalize(this);
         }
 
+        partial void PartialsInnerDispose();
```
Comment (joaompneves) [inline WebViewControl/WebView.cs:198]:
```suggestion
        partial void ExtraDispose();
```

File: WebViewControl/WebView.cs [hunk 2]
```diff
@@ -138,22 +154,34 @@ public partial class WebView : IDisposable {
...
-                chromium.FocusHandler = new InternalFocusHandler(this);
-            }
-
-            EditCommands = new EditCommands(chromium);
-
-            disposables = new IDisposable[] {
-                chromium,
-                AsyncCancellationTokenSource
-            };
+        private void DetachChromiumHandlers() {
```
Comment (joaompneves) [inline WebViewControl/WebView.cs:159]: I dont see a reason for having this... the reports dont show problems here... you need to cut off the roots not the leafs

**Review by joaompneves — CHANGES_REQUESTED** [review]

**Comment by joaompneves** [comment]
Are there any relevant changes in the PR?

**Comment by brunocunhasilva** [comment]
> 

The relevant changes are the Update of CefGlue.Avalonia and the bump of the WebViewControl.Avalonia version

**Comment by brunocunhasilva** [comment]
Replaced the PR with this other one that only updates CefGlue and bumps the version
https://github.com/OutSystems/WebView/pull/254

**Pull request closed without merging**